l_ext_finger_controller:
  controller:
    module: joint_position_controller
    package: bender_arm_control
    type: JointPositionController
  joint_name: l_ext_finger_joint
  joint_speed: 1.0
  joint_compliance_slope: 128
  joint_compliance_margin: 0
  joint_compliance_punch: 0
  joint_torque_limit: 1.0
  torque_enable_init: False
  enable_pid: True
  p_gain: 0.8
  i_gain: 0.5 
  d_gain: 0.0
  i_clamp: 1.0
  motor:
    id: 7
    init: 740
    max: 875
    min: 735

l_int_finger_controller:
  controller:
    module: joint_position_controller
    package: bender_arm_control
    type: JointPositionController
  joint_name: l_int_finger_joint
  joint_speed: 1.0
  joint_compliance_slope: 128
  joint_compliance_margin: 0
  joint_compliance_punch: 0
  joint_torque_limit: 1.0
  torque_enable_init: False
  enable_pid: True
  p_gain: 0.8
  i_gain: 0.5 
  d_gain: 0.0
  i_clamp: 1.0
  motor:
    id: 8
    init: 455
    max: 320
    min: 460